### PR TITLE
move to peer

### PR DESCRIPTION
**Pull request by kramarmm**

### CODE REVIEW

**Comment by max-mykhailenko** [comment]
Are you check build size after moving modal to peer?

**Comment by kramarmm** [comment]
25.5 kB но здесь наверное лучше с "^1.1.0"?

**Comment by kramarmm** [comment]
25.5 kB

**Comment by max-mykhailenko** [comment]
You should also change https://github.com/max-mykhailenko/react-pure-datepicker/blob/master/webpack.config.js#L37

**Comment by max-mykhailenko** [comment]
And check your current build if it contains `modal` code

**Comment by kramarmm** [comment]
when we install datepicker to our project, it will search for modal dependency in existed dep list.
But when we build datepicker separate (in min.js) it should install modal package?
So build size wont change?

**Comment by kramarmm** [comment]
checked.
current build doesnt contains modal code.

File: webpack.config.js [hunk 1]
```diff
@@ -2,11 +2,19 @@ const path = require('path');
...
-  commonjs: 'react',
-  amd: 'react',
+const externals = {
+  react: {
+    root: 'React',
+    commonjs2: 'react',
+    commonjs: 'react',
+    amd: 'react',
+  },
+  ['react-pure-modal']: {
```
Comment (max-mykhailenko) [inline webpack.config.js:12]: можно без квадратных скобок, но это не критично

**Pull request merged**